packages/components/psammead-copyright/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 3.0.24 | [PR#4561](https://github.com/bbc/psammead/pull/4561) Bump dependencies for @bbc/psammead-styles |
```

Comment:
You add the version to the changelog but the package `psammead-copyright` isn't bumped to use 3.0.24?

Reply:
Resolved in 16595e94eab89fc996c3f647c9772a17568fc783